lib/cisco_node_utils/cmd_ref/itd_device_group.yaml

```diff
@@ -0,0 +1,83 @@
+# itd_device_group
+---
+_exclude: [N3k, N5k, N6k, ios_xr]
+
+_template:
+  get_command: "show running-config all | section itd"
+  get_context:
+      - '/^itd device-group <name>/'
+      - '(?)/^node <ntype> <nname>/'
```

Comment:
@glennmatthews Have you and Sai been able to sync on this point?  I don't have a strong opinion here, but I would favor less duplication of YAML entries.